packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -173,8 +197,10 @@ export class VaTextInput {
           inputmode={inputMode}
           maxlength={this.maxlength}
         />
-        {this.maxlength && this.value.length >= this.maxlength && (
-          <small aria-live="polite">(Max. {this.maxlength} characters)</small>
+        {this.maxlength && this.value?.length >= this.maxlength && (
+          <small aria-live="polite">
+            {i18next.t('max-chars', { length: this.maxlength })}
+          </small>
```

Comment:
The optional chaining here is to fix a bug I noticed in storybook where the component doesn't render due to the `value` prop not being set:

![image](https://user-images.githubusercontent.com/2008881/162083404-b42315f8-f49d-49b6-81f4-579ea0f6e861.png)

---

The `i18next` usage is just an example of how we can pass values into the translation string.